autrainer/metrics/sed_metrics.py

```diff
@@ -1,32 +1,246 @@
+import hashlib
+import threading
+from typing import Dict, List
+
 import numpy as np
-from sed_eval.sound_event import EventBasedMetrics
+from sed_eval.sound_event import EventBasedMetrics, SegmentBasedMetrics
+
+from ..datasets.utils.target_transforms.sed_encoder import SEDEncoder
```

Comment:
Best import this from package, i.e. `from autrainer.datasets.utils.target_transforms import SEDEncoder`